frontend/components/hint.jsx

```diff
@@ -75,16 +75,15 @@ class Hint extends React.Component {
     const { show } = this.state;
 
     let hintText;
-    switch (hint) {
+    switch (parseInt(hint)) {
     case 1: hintText = "Each table has a unique set of letters. It seems odd that there isn't a table 26 though."; break;
     case 2: hintText = "If it's not written in ink...then maybe it's white space?"; break;
     case 3: hintText = "Erica's looking for a clue with nine letters? Aren't there nine ingredients, each with their own description?"; break;
     case 4: hintText = "My Chinese name? How in the world does it have anything to do with these icons? Maybe if get the tones all wrong..."; break;
     case 5: hintText = "The paper looks oddly familiar. Did we take a picture with it at the photobooth?"; break;
     case 8: hintText = "Jay probably would've taken the shortest path possible from his car to the sweetheart table."; break;
     }
-
-    if (hintText) {
+    if (hintText != undefined) {
```

Comment:
is this change needed?

also for future reference, we should always use triple equals operator (`===` or `!==`)

Reply:
I guess it isn't if we changed the numbers to strings